build_tools/ci/cpu_comparison/run.py

```diff
@@ -101,12 +125,12 @@ class MatmulFullBias(BaseMatmul):
     A test of the form matmul(A,B) + C where A:MxK, B:KxN, C:MxN
     """
 
-    def __init__(self, M, N, K, input_type, acc_type):
-        super().__init__(M, N, K, input_type, acc_type)
+    def __init__(self, M, N, K, input_type, acc_type, run_on_target=["npu1_4col"]):
```

Comment:
In order to have minimal changes in the file overall - I have chosen to keep `npu1_4col` as the default target device to run the instance on.

Therefore, while registering the instance test I had to change just one of the instance `Vanilla Matmul 32x32x32` to make it run on `npu4` as well.